server.py

```diff
@@ -34,7 +35,7 @@ def add_service_and_version(_, __, event_dict):
     event_dict['version'] = __version__
     return event_dict
 
-logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
+logger_initial_config(service_name='sdx-sequence')
```

Comment:
I think you've lost the log level here?

See https://github.com/ONSdigital/sdx-receipt-rrm/pull/37/files